sdk/aqueduct/flow_run.py

```diff
@@ -64,6 +68,30 @@ def describe(self) -> None:
             assert param_op.spec.param is not None, "Artifact is not a parameter."
             print("* " + param_op.name + ": " + param_op.spec.param.val)
 
+    def artifact(
+        self, artifact_name: str
+    ) -> Union[TableArtifact, MetricArtifact, CheckArtifact, ParamArtifact, None]:
+        """Gets the Artifact from the flow based on artifact_name"""
+        flow_run_dag = self._dag
+
+        for artifact_from_workflow in list(flow_run_dag.artifacts.values()):
+            if artifact_from_workflow.name == artifact_name:
+                assert isinstance(artifact_from_workflow, Artifact)
+                artifact_DAG = artifact_from_workflow
+
+        if artifact_DAG is None:
+            raise ArtifactNotFoundException("The artifact name provided does not exist.")
+        elif artifact_DAG.spec.table is not None:
+            return TableArtifact(self._api_client, self._dag, artifact_DAG.id, True)
+        elif artifact_DAG.spec.float is not None:
+            return MetricArtifact(self._api_client, self._dag, artifact_DAG.id, True)
+        elif artifact_DAG.spec.bool is not None:
+            return CheckArtifact(self._api_client, self._dag, artifact_DAG.id, True)
+        elif artifact_DAG.spec.jsonable is not None:
+            return ParamArtifact(self._api_client, self._dag, artifact_DAG.id, True)
+
```

Comment:
nit: better to use `get_artifact_type()` in these if-else statements, since its more readable and catches any non-valid types.